lib/common/Makefile.am

```diff
@@ -110,7 +110,9 @@ libcrmcommon_test_la_LDFLAGS	= $(libcrmcommon_la_LDFLAGS) -rpath $(libdir) $(LDF
 # disable all builtins.  Older versions of GCC (at least, on RHEL7) will still emit
 # replacement code for strdup (and possibly other functions) unless -fno-inline is
 # also added.
-libcrmcommon_test_la_CFLAGS	= $(libcrmcommon_la_CFLAGS) -DPCMK__UNIT_TESTING -fno-builtin -fno-inline
+libcrmcommon_test_la_CFLAGS	= $(libcrmcommon_la_CFLAGS) -DPCMK__UNIT_TESTING \
+					-fno-builtin -fno-inline \
+					-U_FORTIFY_SOURCE
```

Comment:
Interesting. I'm not really sure what tests that refers to -- probably a lot has changed since 2016. I get the gist about flag combinations in replacements.

Reply:
T191 is strictly about configure.ac, so tests like AC_COMPILE_IFELSE()